client/wfprev-war/src/main/angular/src/app/utils/constants.ts

```diff
@@ -22,6 +22,10 @@ export const Messages = {
   activityDeletedFailure: 'Activity Deleted Failed',
   fileUploadSuccess: 'File Uploaded Successfully',
   fileUploadFailure: 'File Uploaded Failed',
+  evaluationCriteriaCreatedSuccess: 'Evaluation Criteria Created Successfully.',
+  evaluationCriteriaCreatedFailure: 'Evaluation Criteria Created Failed.',
```

Comment:
This should be "Evaluation Criteria _Creation_ Failed"

Reply:
updated